proxy/predeployed/scripts/generate_package_version.py

```diff
@@ -22,14 +23,21 @@ def calculate_version(release_version):
     else:
         return main_part + '.dev' + str(abs(hash(tail_part)))
 
+def getVersion():
+    version_key = 'VERSION'
+    if os.environ.get(version_key):
+        return os.environ.get(version_key)
+
+    try:
+        tag = subprocess.run("git describe --tags", shell=True, check=True, capture_output=True)
+        return tag.stdout.decode('utf-8')[:-1]
+    except:
+        with open('../../VERSION') as f:
```

Comment:
I think we need to make it with try also - if no VERSION there - exit with message